src/assets/css/style.css

```diff
@@ -714,3 +714,24 @@ audio.comfy-audio.empty-audio-widget {
 .p-tree-node-content {
   padding: var(--comfy-tree-explorer-item-padding) !important;
 }
+
+.doc-node {
```

Comment:
Please move this to corresponding vue component.

Reply:
It took a good bit of fighting to make sure the template would actually update, but I agree do with migrating.
- Watching canvas is no longer viable after #1372, but for for the best. Discerning the best alternative was the primary blocker.